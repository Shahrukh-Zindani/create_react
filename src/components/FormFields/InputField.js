import React from 'react'

const InputField = ({ onChange, value }) => (
  <input onChange={onChange} value={value} />
)

export default InputField
