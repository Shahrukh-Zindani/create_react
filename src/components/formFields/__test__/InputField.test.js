import React from 'react'
import renderer from 'react-test-renderer'
import InputField from '../InputField'

describe('Input Field tests', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(
      <InputField />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
