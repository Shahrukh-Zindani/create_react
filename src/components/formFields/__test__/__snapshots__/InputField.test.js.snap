// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Field tests matches snapshot 1`] = `
<input
  onChange={[Function]}
  value=""
/>
`;
